src/dynamics/impulse_joint.rs

```diff
@@ -64,6 +86,19 @@ impl RawImpulseJointSet {
         self.map(handle, |j| j.data.limits[axis as usize].max)
     }
 
+    /// Enables and sets the joint limits
+     pub fn setJointLimits(
+        &mut self,
+        handle: FlatHandle,
+        axis: RawJointAxis,
+        min: f32,
+        max: f32,
+    ) {
+        self.map_mut(handle, |j| {
+            j.data.set_limits(JointAxis::from(axis).into(), [min, max]);
```

Comment:
```suggestion
            j.data.set_limits(axis.into(), [min, max]);
```

Reply:
Hi @sebcrozet - this change has now been made. Thanks